tests/framework/integration.go

```diff
@@ -155,3 +158,41 @@ func (c integrationClient) Compact(rev int64, o config.CompactOption) (*clientv3
 	}
 	return c.Client.Compact(ctx, rev, clientOpts...)
 }
+
+func (c integrationClient) Status() ([]*clientv3.StatusResponse, error) {
+	endpoints := c.Client.Endpoints()
+	var resp []*clientv3.StatusResponse
+	for _, ep := range endpoints {
+		status, err := c.Client.Status(context.Background(), ep)
+		if err != nil {
+			return nil, err
+		}
+		resp = append(resp, status)
+	}
+	return resp, nil
+}
+
+func (c integrationClient) HashKV(rev int64) ([]*clientv3.HashKVResponse, error) {
+	endpoints := c.Client.Endpoints()
+	var resp []*clientv3.HashKVResponse
+	for _, ep := range endpoints {
+		hashKV, err := c.Client.HashKV(context.Background(), ep, rev)
+		if err != nil {
+			return nil, err
+		}
+		resp = append(resp, hashKV)
+	}
+	return resp, nil
+}
+
+func (c integrationClient) Health() error {
+	cli := healthpb.NewHealthClient(c.Client.ActiveConnection())
+	resp, err := cli.Check(context.TODO(), &healthpb.HealthCheckRequest{})
+	if err != nil {
+		return err
+	}
+	if resp.Status != healthpb.HealthCheckResponse_SERVING {
+		return fmt.Errorf("status expected %s, got %s", healthpb.HealthCheckResponse_SERVING, resp.Status)
+	}
+	return nil
+}
```

Comment:
The integration health implementation is different than e2e one. It's not a big issue but just want to call it out.